disperser/dataapi/operator_handler.go

```diff
@@ -84,23 +83,36 @@ func (o *OperatorList) GetID(address string) (core.OperatorID, bool) {
 	return id, exists
 }
 
-func NewOperatorHandler(logger logging.Logger, metrics *Metrics, chainReader core.Reader, chainState core.ChainState, indexedChainState core.IndexedChainState, subgraphClient SubgraphClient) *OperatorHandler {
+func NewOperatorHandler(logger logging.Logger, metrics *Metrics, chainReader core.Reader, chainState core.ChainState, indexedChainState core.IndexedChainState, subgraphClient SubgraphClient) (*OperatorHandler, error) {
+	// Determine valid set of quorum IDs at startup
+	currentBlock, err := chainReader.GetCurrentBlockNumber(context.Background())
+	if err != nil {
+		return nil, err
+	}
+	quorumCount, err := chainReader.GetQuorumCount(context.Background(), uint32(currentBlock))
```

Comment:
If the onchain state changes for quorum count, it seems still correct, just incomplete (e.g. it gives results from quorum 0 to 2, and missing for quorum 3)

Reply:
meaning it requires a restart if quorum is added? yes

Reply:
more about what will go wrong if it hasn't restarted to catch up the onchain state